assets/css/amp-stories.css

```diff
@@ -50,3 +50,7 @@ amp-story-grid-layer[template="fill"] .wp-block-image { margin: 0; }
 .is-style-half-rounded {
 	border-radius: 4px 4px 26px 26px;
 }
+
+amp-story-grid-layer amp-img > img {
+	object-position: inherit;
```

Comment:
This is apparently needed because the `<img>` didn't seem to inherit the `object-position` value without this.